.browserslistrc

```diff
@@ -0,0 +1,4 @@
+# Browsers that we support
+
+defaults
```

Comment:
can we be more explicit? I don't really want to have to look at the documentation for the default. also more comments in this file would be really helpful, to explain its context